### PR TITLE
Added myextension: trackFlight!

### PR DESCRIPTION
**Pull request by cojmeister**
## Description

An extension which allows users to track flight status from raycast! See airport, time and status!

## Screencast

trackflight-1
trackflight-2
trackflight-3

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @cojmeister 👋 

Is there a place I can DM you? Either in our [Slack](https://www.raycast.com/community) or on [Twitter](https://twitter.com/pertikaer) 😊

**Comment by cojmeister** [comment]
Hi @pernielsentikaer I DMd you on twitter

**Comment by cojmeister** [comment]
Given that this feature exists in the newest version, will it be merged regardless?
@pernielsentikaer

File: extensions/trackflight/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "Data",
+    "Productivity",
+    "Web"
+  ],
+  "commands": [
+    {
+      "name": "trackFlight",
+      "title": "Track Flight Number:",
+      "subtitle": "Just type in your flight number",
+      "description": "Just fill in will your flight number and we'll get your data!!",
```
Comment (pernielsentikaer) [inline extensions/trackflight/package.json:19]:
```suggestion
      "title": "Track Flight",
      "description": "Track your flight directly from Raycast by flight number",
```

File: extensions/trackflight/src/trackFlight.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          popToRoot({ clearSearchBar: true });
+        }, 500);
+      }
+    }
+    setFlightDate("today");
+  }, []); // only run this effect once when the component mounts
+
+  function actionCommands() {
+    return (
+      <ActionPanel title="#1 in raycast/extensions">
```
Comment (pernielsentikaer) [inline extensions/trackflight/src/trackFlight.tsx:36]:
```suggestion
      <ActionPanel title="Flight Date">
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Sure we can 😊

Left a few suggestions for you, checking again after this 😊

**Comment by cojmeister** [comment]
> Hi 👋 
> 
> 
> 
> Sure we can 😊
> 
> 
> 
> Left a few suggestions for you, checking again after this 😊

Changes are done 😊 🎉

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**